generation/WinSDK/enums.json

```diff
@@ -31894,6 +31763,224 @@
     },
     "members": [],
     "uses": []
+  },
+  {
+    "type": "uint",
+    "name": "NOTIFY_ICON_INFOTIP_FLAGS",
+    "autoPopulate": {
+      "filter": "NIIF_",
+      "header": "shellapi.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "struct": "NOTIFYICONDATAW",
+        "field": "dwInfoFlags"
+      },
+      {
+        "struct": "NOTIFYICONDATAA",
+        "field": "dwInfoFlags"
+      }
+    ]
+  },
+  {
+    "type": "uint",
+    "name": "NOTIFY_ICON_STATE",
+    "autoPopulate": {
+      "filter": "NIS_",
+      "header": "shellapi.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "struct": "NOTIFYICONDATAW",
+        "field": "dwState"
+      },
+      {
+        "struct": "NOTIFYICONDATAA",
+        "field": "dwState"
+      }
+    ]
+  },
+  {
+    "name": "PAN_FAMILY_TYPE",
+    "autoPopulate": {
+      "filter": "PAN_FAMILY_",
+      "header": "wingdi.h"
+    },
+    "members": [
+      {
+        "name": "PAN_FAMILY_ANY",
+        "value": 0
+      },
+      {
+        "name": "PAN_FAMILY_NO_FIT",
+        "value": 1
+      }
+    ],
+    "uses": []
```

Comment:
All of the PAN enums are missing uses. Is this on purpose?

Reply:
Not on purpose! Just added them!